src/block/block_open.c

```diff
@@ -210,17 +209,19 @@ __wt_block_open(WT_SESSION_IMPL *session, const char *filename, WT_BLOCK_FILE_OP
     else if (WT_STRING_MATCH("sequential", cval.str, cval.len))
         LF_SET(WT_FS_OPEN_ACCESS_SEQ);
 
+    if (fixed)
+        LF_SET(WT_FS_OPEN_FIXED);
```

Comment:
This seems unrelated? Is this part of some other ticket? (Also should be alphabetized.)

Reply:
This wasn't an unrelated change, tiered storage opens fixed filename paths. (I have no idea if that's right or not, but that's what it does from before, see https://github.com/wiredtiger/wiredtiger/blob/develop/src/tiered/tiered_opener.c#L73).

I'm not sure what you mean by "should be alphabetized"? Can you explain further, please?

Reply:
By "alphabetized" I mean that these two lines should go below the ones that set WT_FS_OPEN_DIRECTIO a few lines down.

Reply:
That would mean moving the lines into the middle of 3 clauses, all dealing with the `readonly` flag, that's why I did it the way I did.

(The fact we're passing 3 booleans into this function is flawed; we should rework this flag handling entirely.)